Lib/test/test_logging.py

```diff
@@ -4347,6 +4347,18 @@ def test_queue_listener(self):
         self.assertTrue(handler.matches(levelno=logging.CRITICAL, message='6'))
         handler.close()
 
+    @unittest.skipUnless(hasattr(logging.handlers, 'QueueListener'),
```

Comment:
I'm surprised but when is `QueueListener` not available?

Reply:
Great question. No idea. I just matched the surrounding tests 😆

Reply:
@vsajip any idea why we have this? is this still relevant or was it a relicate from the past? because the class seems to be unconditionally defined

Reply:
It used to be that `threading` wasn't always available - at one time (almost 15 years ago!), `QueueListener` wasn't defined if `threading` couldn't be imported. But that's a _long_ time ago ... these `skipUnless` decorators can be removed now, I think.

Reply:
Updated. Thanks!